dynamic-plugins/imports/package.json

```diff
@@ -16,24 +16,24 @@
     "tar": "6.2.1"
   },
   "peerDependencies": {
-    "@janus-idp/backstage-plugin-3scale-backend": "1.5.15",
-    "@janus-idp/backstage-plugin-aap-backend": "1.6.15",
-    "@janus-idp/backstage-plugin-acr": "1.4.13",
-    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.9",
-    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.11",
-    "@janus-idp/backstage-plugin-keycloak-backend": "1.9.12",
-    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.10",
-    "@janus-idp/backstage-plugin-ocm": "4.1.8",
+    "@janus-idp/backstage-plugin-3scale-backend": "1.5.16",
+    "@janus-idp/backstage-plugin-aap-backend": "1.6.16",
+    "@janus-idp/backstage-plugin-acr": "1.4.14",
+    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.4.10",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.4.12",
+    "@janus-idp/backstage-plugin-keycloak-backend": "1.9.13",
+    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.6.11",
+    "@janus-idp/backstage-plugin-ocm": "4.1.9",
     "@janus-idp/backstage-plugin-ocm-backend": "4.0.9",
```

Comment:
```suggestion
    "@janus-idp/backstage-plugin-ocm-backend": "4.0.10",
```

Reply:
> 

This is the plugin with issues that's causing e2e breakage.  See https://github.com/janus-idp/backstage-showcase/pull/1735